runtime/metricsview/ast.go

```diff
@@ -372,6 +372,21 @@ func (a *AST) resolveMeasure(qm Measure, visible bool) (*runtimev1.MetricsViewSp
 		}, nil
 	}
 
+	if qm.Compute.PercentOfTotal != nil {
+		m, err := a.lookupMeasure(qm.Compute.PercentOfTotal.Measure, visible)
+		if err != nil {
+			return nil, err
+		}
+
+		return &runtimev1.MetricsViewSpec_MeasureV2{
+			Name:               qm.Name,
+			Expression:         fmt.Sprintf("%s*100/%f", a.dialect.EscapeIdentifier(m.Name), qm.Compute.PercentOfTotal.Total),
+			Type:               runtimev1.MetricsViewSpec_MEASURE_TYPE_DERIVED,
```

Comment:
1. We are calculating the measure value itself in a subquery since it might be used for comparison columns as well. So adding the expression here will recompute it wont it?
2. Update to be safer.

Reply:
Okay, if you think it will mostly be queried in requests that also query the plain measure value, then I agree the derived approach is better

Reply:
Ya. I would assume all instances of totals in dashboard right now can be replaced with this.